Classes/Aggregate/BackendPreviewAggregate.php

```diff
@@ -180,9 +208,17 @@ protected function getProcessedData(array \$databaseRow, array \$processedTcaCol
                 continue;
             }
             \$processedRow[\$field] = [];
+            \$i = 0;
             foreach (\$config['children'] as \$child) {
                 \$processedRow[\$field][] = \$this->getProcessedData(\$child['databaseRow'], \$child['processedTca']['columns']);
+                \$files = \$this->getFalFiles(\$processedRow[\$field][\$i]['uid'], 'tt_content', 'tx_' . \$processedRow[\$field][\$i]['CType'][0]);
```

Comment:
You have to validate the $processedRow if the desired keys are available. Otherwise this code will trigger notices